hivemind/server/__init__.py

```diff
@@ -1,8 +1,10 @@
+from __future__ import annotations
 import multiprocessing as mp
 import multiprocessing.synchronize
 import threading
 from contextlib import contextmanager
 
+import numpy as np
```

Comment:
Do we really need numpy here just for randint generation?

Reply:
switched to random